source/common/res/features/budget-category-info/main.js

```diff
@@ -160,7 +160,8 @@
           // when this is moved to the webpack codebase, hopefully we can just listen to onRouteChanged
           // but until then we'll just have to keep adding stuff to this check that should trigger
           // budget-category related features to update
-          if (changedNodes.has('navlink-budget active') ||
+          if ((changedNodes.has('nav-main') && ynabToolKit.shared.getCurrentRoute().includes('budget')) ||
+            changedNodes.has('budget-table-row is-sub-category') ||
```

Comment:
This handles moving from one sub-category to another.... this is probably a super-set of the 'bugdet-inspector-goals' node, so it can probably be removed, but I don't think it matters. Worst case scenario is that we run the update-rows function needlessly sometimes, which I don't think would happen.